AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -145,6 +149,10 @@ - (void)updateCurrentRangeWithMode:(ASLayoutRangeMode)rangeMode
     _didUpdateCurrentRange = YES;
```

Comment:
The answer is: I'm not sure.

We likely need to audit this system and bring visibilityDepth knowledge to the range controller post 2.0